### PR TITLE
manifest: nrfxlib: Update SDC and MPSL.

### PR DESCRIPTION
**Pull request by ibaz-nordic**

### CODE REVIEW

File: doc/nrf/releases/release-notes-changelog.rst
Comment (rugeGerritsen) [inline doc/nrf/releases/release-notes-changelog.rst:49]: Discussed offline. The update the release notes should be removed

**Review by rugeGerritsen — CHANGES_REQUESTED** [review]

**Comment by PerMac** [comment]
87/112 nrf5340dk_nrf5340_cpuapp  zigbee/light_switch/sample.zigbee.light_switch.fota_and_multiprotocol FAILED Build failure (build) is failing due to `/tools/ncs/toolchains/v2.1.0/opt/zephyr-sdk/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr/zephyr_pre0.elf section 'noinit' will not fit in region 'SRAM'`
 @tomchy @sebastiandraus I guess you might want to take a look at this. Do you think the sample can be optimized to adopt to the changes introduced with updating of SDC and MPSL?

**Comment by ibaz-nordic** [comment]
> 87/112 nrf5340dk_nrf5340_cpuapp zigbee/light_switch/sample.zigbee.light_switch.fota_and_multiprotocol FAILED Build failure (build) is failing due to `/tools/ncs/toolchains/v2.1.0/opt/zephyr-sdk/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr/zephyr_pre0.elf section 'noinit' will not fit in region 'SRAM'` @tomchy @sebastiandraus I guess you might want to take a look at this. Do you think the sample can be optimized to adopt to the changes introduced with updating of SDC and MPSL?

With the slightly smaller SDC, the test survives.

You might want to consider reducing ([CONFIG_BT_MAX_CONN](https://github.com/nrfconnect/sdk-nrf/blob/main/samples/zigbee/light_switch/child_image/multiprotocol_rpmsg/boards/nrf5340dk_nrf5340_cpunet_fota.conf#L19)) to reduce memory. I don't think you need that many.

**Review by rugeGerritsen — APPROVED** [review]

**Review by bugo-nordic — APPROVED** [review]

File: subsys/bluetooth/controller/hci_driver.c
Comment (rugeGerritsen) [inline subsys/bluetooth/controller/hci_driver.c:624]: We need to be able to set the number of advertising sets when not supporting the peripheral
```suggestion
	if (IS_ENABLED(CONFIG_BT_BROADCASTER)) {
```

**Review by rugeGerritsen — CHANGES_REQUESTED** [review]
I would be happy if you could change the commit message to:
```
Bluetooth: Controller: Don't set adv set count if not supported

Setting the number of advertisers for a build configuration that does not support the peripheral role will fail.
```

**Review by joerchan — CHANGES_REQUESTED** [review]
This PR was first pointing to nrfxlib#853, which was merged without this manifest PR ever being green, or merged.

This is currently blocking others from merging to nrfxlib because 853 has a regression and the Bluetooth test-cases are red.

I am therefore reverting the changes of 853, and blocking this PR until that has been done.
This PR needs to be rebased on top of https://github.com/nrfconnect/sdk-nrf/pull/9179/files once that is merged.

**Comment by ibaz-nordic** [comment]
Finally passed CI.

Code owners - please let me know if you have any concerns.
Appreciate it if we could merge this ASAP as it has been blocked for a while (due to CI failure).

**Review by jori-nordic — APPROVED** [review]

**Comment by jori-nordic** [comment]
> Finally passed CI.
> 
> Code owners - please let me know if you have any concerns. Appreciate it if we could merge this ASAP as it has been blocked for a while (due to CI failure).

tag them directly

**Review by jciupis — APPROVED** [review]

File: west.yml
Comment (rlubos) [inline west.yml:119]: Please update SHA: `688cdced97ff20d507def2d17d42d20654968758`

**Review by rlubos — APPROVED** [review]

**Review by rugeGerritsen — APPROVED** [review]

**Review by canisLupus1313 — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]
lgtm

**Pull request merged**
